### DISCUSSION TITLE
32-bit Windows build?

### DISCUSSION THREAD

**Discussion by JBetz** [cannerycoders/fluidsynth.clap#2] [Q&A]
Is it possible to make a 32-bit build of the DLL for Windows?

64-bit seems to be the de facto standard for CLAP, but since FluidSynth support 32-bit, a 32-bit build of fluidsynth.clap could still be useful.

I do a lot of development on 32-bit Windows, so would definitely use and appreciate it.

**Comment by dbadb** [comment]
I am curious what/how many 32 bit windows machines are still around and available.

**Reply by JBetz** [reply]
For me it's a software issue, not hardware.

Dolphin Smalltalk which is 32-bit and probably not going to be upgraded anytime soon. And I use Dolphin to develop music software. Long term, I plan to run a 64-bit executable in a separate process to deal with it, but having something 32-bit to play around with would be useful for the interim.

If 32-bit builds are non-trivial then I understand not supporting it. But I wanted to ask in the chance it was just a matter of tweaking a build script.
